code/datums/components/mood.dm

```diff
@@ -280,14 +282,14 @@
 	qdel(event)
 	update_mood()
 
-/datum/component/mood/proc/remove_temp_moods(var/admin) //Removes all temp moods
+/datum/component/mood/proc/remove_temp_moods() //Removes all temp moods
 	for(var/i in mood_events)
 		var/datum/mood_event/moodlet = mood_events[i]
 		if(!moodlet || !moodlet.timeout)
 			continue
 		mood_events -= moodlet.category
 		qdel(moodlet)
-		update_mood()
+	update_mood()
```

Comment:
This is mostly a change to improve how well this proc runs. We don't have anything that would require updating mood for every single removed moodlet. At least not currently.

Reply:
good god how did i miss this, thanks.